abayo-luc/hello-books
# #164426639 User Sign In

#### Description
Add user sign in functionality, by generating``` json web token``` when user provides valid email and password
#### Type of change
- [x] Add new feature
#### How Has This Been Tested?
- clonse the repose
- checkout the branch ```git checkout ft-user-login-164426639```
- update the gem ```bundle update```
- run authomated test ```bundle exc rspec``` 
- access the end point ```POST /api/v1/users/login``` and provided ```{email:"", password:""}``` params. 
- The response should be an object with ```token```
- Supply the token to access the following endpoint:
- [ ] ```GET /api/v1/profiles```
- [x] ```PUT /api/v1/profiles/:id```
- [x] ```GET /api/v1/profiles/:id```
#### PivotalTracker ID
#164426639
